specs/WebResourceRequested-CustomScheme.md

```diff
@@ -341,6 +346,26 @@ interface ICoreWebView2CustomSchemeRegistration : IUnknown {
   HRESULT SetAllowedOrigins(
     [in] UINT32 allowedOriginsCount,
     [in] LPCWSTR* allowedOrigins);
+
+  // Set this to `true` if the URLs with this custom scheme will have an
+  // authority component (a host for custom schemes).
+  // eg: custom-scheme-with-authority://host/path - Set to `true`
+  //     custom-scheme-without-authority:path - Set to `talse`
+  // When this is set to `true`, the URLs with this scheme will be
+  // interpreted as they have a
+  // [scheme and host](https://html.spec.whatwg.org/multipage/origin.html#concept-origin-tuple) 
+  // origin similar to an http URL. Note that the port and user
+  // information are never included in the computation of origins for
+  // custom schemes. If you set `HasAuthorityComponent` to `true` and 
+  // use a URL with this scheme that does not have
+  // an authority component, the behavior of such URLs will be undefined.
+  // `false` by default. 
```

Comment:
Is false the better default? Is it the safer default or what we expect to be more common?

Reply:
It's the safer default as officially custom scheme URLs have opaque origins. Allowing URIs with host origins improves the usability of custom schemes, but I think that should be opted in.